Tools/CISupport/ews-build/steps.py

```diff
@@ -2891,11 +2892,12 @@ def send_email_for_new_build_failure(self):
                 logs = logs.replace('&', '&amp;').replace('<', '&lt;').replace('>', '&gt;')
                 email_text += '\n\nError lines:\n\n<code>{}</code>'.format(logs)
             email_text += '\n\nTo unsubscribe from these notifications or to provide any feedback please email aakash_jain@apple.com'
-            self._addToLog('stdio', 'Sending email notification to {}'.format(change_author))
+            yield self._addToLog('stdio', f'Sending email notification to {change_author}')
             send_email_to_patch_author(change_author, email_subject, email_text, patch_id or self.getProperty('github.head.sha', ''))
         except Exception as e:
-            print('Error in sending email for new build failure: {}'.format(e))
+            yield self._addToLog('stdio', f'Error in sending email for new build failure: {e}')
 
+    @defer.inlineCallbacks
```

Comment:
We have to `yield` calls to `send_email_for_preexisting_build_failure` now too, right? Which is, I think, the reason we haven't done this yet